<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2125</width>
    <height>1076</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>나스닥100 호가창 예제 (확장형)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QFrame" name="frame_left">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_left">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_top">
         <layout class="QHBoxLayout" name="horizontalLayout_top">
          <item>
           <widget class="QLabel" name="label_market">
            <property name="text">
             <string>나스닥100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_price">
            <property name="text">
             <string>24,590.50</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_change">
            <property name="text">
             <string>▲ 38.75  -0.16%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_chart">
            <property name="text">
             <string>차트</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_account">
            <property name="text">
             <string>계좌관리</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_setting">
            <property name="text">
             <string>환경설정</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_stocklist">
         <property name="text">
          <string>종목목록</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="table_hoga">
         <column>
          <property name="text">
           <string>MIT</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>매도</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>건수</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>잔량</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>고정</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>잔량</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>건수</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>매수</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>MIT</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_buttons">
         <layout class="QHBoxLayout" name="horizontalLayout_buttons">
          <item>
           <widget class="QPushButton" name="button_sell_res_cancel">
            <property name="text">
             <string>매도예약취소</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_sell_cancel">
            <property name="text">
             <string>매도취소</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_all_cancel">
            <property name="text">
             <string>일괄취소</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_item_cancel">
            <property name="text">
             <string>종목취소</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_buy_cancel">
            <property name="text">
             <string>매수취소</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_buy_res_cancel">
            <property name="text">
             <string>매수예약취소</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>전체</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTableWidget" name="tableWidget_6">
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>잔고</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>미체결</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTableWidget" name="tableWidget_5">
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>예약</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_center">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="7" column="0" colspan="3">
        <widget class="QTableWidget" name="table_trades">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>시간</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>가격</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>수량</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>체결현황</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2" colspan="2">
        <widget class="QTreeWidget" name="treeWidget_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QTableWidget" name="tableWidget_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QTableWidget" name="table_stocklist">
         <property name="columnCount">
          <number>4</number>
         </property>
         <column>
          <property name="text">
           <string>종목명</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>코드</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>현재가</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>전일대비</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>원격지원</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>통합주문</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>잔고목록</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>거래내역</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>오버신청</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>모의신청</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>환경설정</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>1:1문의</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>공지사항</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
